inf-terraform-aws/files/Makefile

```diff
@@ -4,7 +4,11 @@ DESCRIPTION              := The '$(NAME)' is a is a prototype for an ODS quickst
 PWD                      := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 GEMS_HOME                ?= $(PWD)/vendor/bundle
 INSTALL_REPORT_HOME      := ./reports/install
-SHELL                    := /usr/bin/env bash
+SHELL                    := /usr/bin/env bash -o pipefail
```

Comment:
there are flags in here that are defined in `SHELLFLAGS` already

Reply:
uups ... forgot to remove that while applying your example

Reply:
fixed by https://github.com/opendevstack/ods-quickstarters/pull/687/commits/0e9598753d119645eb0a33abc319841d5b64614e